hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/test/TestSources.java

```diff
@@ -132,6 +136,50 @@ private TestSources() {
             .build();
     }
 
+    /**
+     * Returns a {@link com.hazelcast.jet.pipeline.StreamSource} that emits an
```

Comment:
I don't think you need the fully qualified name here, package is already imported.

Reply:
Done with 27522575900e3e43d31d1615b99d225ea8025c35